src/scripts/autoloads/analytics.gd

```diff
@@ -59,8 +64,16 @@ func _ready():
 	if OS.has_feature("HTML5"):
 		session_id = JavaScript.eval("getSessionId()")
 		user_id = JavaScript.eval("getUserId()")
+		var window := JavaScript.get_interface("window")
+		window.addEventListener("oneHourEvent", one_hour_callback_ref)
 	else:
 		user_id = str(OS.get_unique_id())
+		one_hour_timer = Timer.new()
+		one_hour_timer.wait_time = 3
```

Comment:
hmm, from my understanding of what you are doing... shouldn't this be 3600 seconds instead of 3?

Reply:
oh i changed it to test and forgot to change it back.